src/os_win.cc

```diff
@@ -33,6 +33,63 @@
 
 namespace toit {
 
+static wchar_t* malloced_wide_string(const char* string) {
+  word length = Utils::utf_8_to_16(unsigned_cast(string), strlen(string));
+  wchar_t* result = reinterpret_cast<wchar_t*>(malloc((length + 1) * sizeof(wchar_t)));
+  Utils::utf_8_to_16(unsigned_cast(string), strlen(string), result, length);
+  result[length] = '\0';
+  return result;
+}
+
+
+static char* malloced_narrow_string(const wchar_t* string, word w_length) {
+  word length = Utils::utf_16_to_8(string, w_length, null, 0);
+  char* result = unvoid_cast<char*>(malloc(length + 1));
+  Utils::utf_16_to_8(string, w_length, unsigned_cast(result), length);
+  result[length] = '\0';
+  return result;
+}
+
+static char* malloced_narrow_string(const wchar_t* string) {
+  word length = wcslen(string);
+  return malloced_narrow_string(string, length);
+}
+
+char* OS::get_executable_path() {
+  char* path = _new char[MAX_PATH + 1];
+  auto length = GetModuleFileName(NULL, path, MAX_PATH);
+  path[length] = '\0';
+  return path;
+}
+
+char* OS::get_executable_path_from_arg(const char* source_arg) {
+  wchar_t* w_source_arg = malloced_wide_string(source_arg);
```

Comment:
I find it kinda odd that these have a `w_` prefix and not a `_w` suffix. YMMV.

Reply:
Changed (hopefully consistently) to `_w`.
done.